### ISSUE TITLE
Crash on 3.9.0

### ISSUE THREAD

**Issue by absolute8511** [aerospike/aerospike-server#141]
While testing the newest 3.9.0, the server crashed as below:

```
Jul 22 2016 02:15:53 GMT: INFO (nsup): (thr_nsup.c:1172) {test} nsup start
Jul 22 2016 02:15:53 GMT: INFO (nsup): (thr_nsup.c:1224) {test} deleting set test
Jul 22 2016 02:15:53 GMT: INFO (nsup): (thr_nsup.c:1224) {test} deleting set test_list2
Jul 22 2016 02:15:53 GMT: INFO (nsup): (thr_nsup.c:1224) {test} deleting set testset
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:181) SIGSEGV received, aborting Aerospike Community Edition build 3.9.0 os el6
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: found 13 frames
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 0: /usr/bin/asd(as_sig_handle_segv+0x3a) [0x4a81e6]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 1: /lib64/libc.so.6(+0x32660) [0x7f7faf6ae660]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 2: /usr/bin/asd(cf_rc_release+0x5) [0x52d030]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 3: /usr/bin/asd(as_fabric_msg_put+0xe) [0x4f33dd]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 4: /usr/bin/asd(rw_request_destroy+0xc9) [0x523b99]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 5: /usr/bin/asd() [0x521779]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 6: /usr/bin/asd(repl_write_handle_ack+0x21a) [0x5219a6]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 7: /usr/bin/asd(rw_msg_cb+0x107) [0x523471]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 8: /usr/bin/asd(fabric_process_read_msg+0x380) [0x4f6552]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 9: /usr/bin/asd(fabric_process_readable+0x3c) [0x4f69af]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 10: /usr/bin/asd(fabric_worker_fn+0x2ee) [0x4f6d1a]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 11: /lib64/libpthread.so.0(+0x7aa1) [0x7f7fb07d9aa1]
Jul 22 2016 02:15:53 GMT: WARNING (as): (signal.c:185) stacktrace: frame 12: /lib64/libc.so.6(clone+0x6d) [0x7f7faf764aad]
```

PS: I am deleting the set and also the migration is running.

**Comment by wchu-citrusleaf** [comment]
Do you have a core file? If yes, please email me and I'll have an ftp site for you to upload to?

Thanks.

**Comment by absolute8511** [comment]
Sorry, the core file is disabled on server.

**Comment by wchu-citrusleaf** [comment]
This a regression introduced in 3.9 as part of -
[AER-4868] - (KVS) Expire-/evict-deletes no longer do duplicate resolution.
The fix is now on the hotfix/3.9 branch, as well as in 3.9.0.1

Thanks.
